### PR TITLE
Remove old annotation leftovers

### PR DESCRIPTION
**Pull request by King-of-Babylon**
I tried this example for our future GTKD exercises and I found some leftovers of the old route annotation way and also I added the code, which is missing to complete the example (I tried it out locally and get some errors e.g. the _entity is missing in the controller route attribute)

### CODE REVIEW

**Review by Isengo1989 — APPROVED** [review]
Thx for the PR :tada:

File: guides/plugins/plugins/framework/store-api/add-store-api-route.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
 use Shopware\Core\Framework\Plugin\Exception\DecorationPatternException;
 use Shopware\Core\Framework\Routing\Annotation\Entity;
 use Shopware\Core\System\SalesChannel\SalesChannelContext;
-use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Attribute\Route;
```
Comment (MalteJanz) [inline guides/plugins/plugins/framework/store-api/add-store-api-route.md:64]: Great that this article is up to date then 🙂 .

But I guess there is still work for many other articles, right? Just search for this use statement in this repository:
https://github.com/search?q=repo%3Ashopware%2Fdocs%20use%20Symfony%5CComponent%5CRouting%5CAnnotation%5CRoute%3B&type=code

I find 17 files that still use this, now outdated, way of annotation 🙈

Comment (King-of-Babylon) [reply guides/plugins/plugins/framework/store-api/add-store-api-route.md:64]: I will have a look, if I could find and fix other files :+1:

Comment (King-of-Babylon) [reply guides/plugins/plugins/framework/store-api/add-store-api-route.md:63]: @MalteJanz @Isengo1989 I found some other places :blush: Pls have a look :blue_heart:

**Comment by Isengo1989** [comment]
@King-of-Babylon thank you :+1: 

The code guidelines from the core get copied from shopware/shopware and will be overwritten. Would be nice if we update it in core as well.

https://github.com/shopware/shopware/blob/trunk/coding-guidelines/core/6.5-new-php-language-features.md

**Pull request merged**